src/widget/wmainmenubar.h

```diff
@@ -34,7 +35,7 @@ class WMainMenuBar : public QMenuBar {
   public:
     WMainMenuBar(QWidget* pParent, UserSettingsPointer pConfig,
                  ConfigObject<ConfigValueKbd>* pKbdConfig);
-
+    void createMenu(std::function<void(QMenu*)>);
```

Comment:
To avoid repition:
```
typedef std::function<void(QMenu*)> FnAddMenu;
```

The typedef does not need to be global, could be restricted to this class.